extensions/wikia/FacebookClient/FacebookClientFactory.php

```diff
@@ -27,7 +27,12 @@ public function connectToFacebook( $wikiaUserId, $fbUserId ) {
 				$status->setResult( false );
 				$status->error( 'fbconnect-error-already-connected' );
 			} else {
```

Comment:
Start adding the business token whenever we create new mappings.  The App ID is already added automatically (has been since we updated the FB code).  Since its a global variable, its not passed in like the business token is.